requirements.txt

```diff
@@ -1,4 +1,3 @@
-configparser>=3.5.0
-fuzzfetch==0.5.7
```

Comment:
Why remove these dependencies?

Reply:
They weren't removed - they now live in `setup.py`, and if you notice the `.` in requirements.txt, when you do a `pip install --user --upgrade -r requirements.txt`, the command will also install `configparser` and `fuzzfetch`.

Reply:
Let me know if you prefer to have them back in, e.g. for pipenv or some of the other 100000 ways of dealing with pip ;-)

Reply:
I was under the impression that both needs to exist. Requirements.txt for the application requirements (version matched) and setup.py for library requirements.